### PR TITLE
LUN-371: Rename CustomModal

### PR DESCRIPTION
**Pull request by charludo**
This pull request belongs to an issue on our [bugtracker](https://issues.tuerantuer.org/secure/RapidBoard.jspa?rapidView=27&view=detail).
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword _LUN_.

### CODE REVIEW

**Comment by charludo** [comment]
Just to be sure, since this is just internal and not something a user would ever know about, no release note should be created, correct?

**Comment by SteffiStuffel** [comment]
> Just to be sure, since this is just internal and not something a user would ever know about, no release note should be created, correct?

correct :+1:

File: src/components/Modal.tsx [hunk 1]
```diff
@@ -15,7 +15,7 @@ const Message = styled(HeadingText)`
   text-align: center;
 `
 
-export interface CustomModalProps {
+export interface ModalProps {
```
Comment (SteffiStuffel) [inline src/components/Modal.tsx:18]: (optional) we try to name prop types just `Props`, but as this is not consistent yet over the project, just fine if you keep this here, too. But just fyi.

**Review by SteffiStuffel — APPROVED** [review]
Not tested, looks good.

File: src/components/Modal.tsx (hunk 1 above)
Comment (charludo) [reply src/components/Modal.tsx]: thanks for letting me know, changed it.

File: src/components/Modal.tsx [hunk 2]
```diff
@@ -28,7 +28,7 @@ export interface CustomModalProps {
 }
 
 // TODO Further adjustments gonna be done with LUN-312
```
Comment (steffenkleinle) [inline src/components/Modal.tsx:30]: Optional: This comment is outdated and could be removed. Would be nice if you could remove it.

**Review by steffenkleinle — APPROVED** [review]
Tested on android

File: src/components/Modal.tsx (hunk 2 above)
Comment (charludo) [reply src/components/Modal.tsx:30]: done

Comment (steffenkleinle) [reply src/components/Modal.tsx:30]: @charludo I think you didn't push it, did you?

Comment (charludo) [reply src/components/Modal.tsx]: oops, sorry. It is now.

File: src/components/Modal.tsx [hunk 3]
```diff
@@ -58,4 +57,4 @@ const Modal = (props: Props): JSX.Element => {
     </ModalSkeleton>
   )
 }
-export default Modal
+eport default Modal
```
Comment (steffenkleinle) [inline src/components/Modal.tsx]: Typo
```suggestion
export default Modal
```

**Pull request merged**